### ISSUE TITLE
Section 5.2.3 -  Can the communicated Intents be profiles of abstract intents?

### ISSUE THREAD

**Issue by gorryfair** [taps-api/drafts#59]
When we worked on the NEAT architecture, it was clear that this type of function could be communicated via the API from the App, but in some cases the App may not wish a detailed spec and simply indicate a profile (naming a collection of the things it needs), allowing a policy to be instantiated that implements the actual control over which candidate paths to allow. For example, “use my corporate network”, “do not bill me for this”, to give very high-level examples that may match to actual requirements about whether public wifi is acceptable. My point here is that I would like to see two things:
** The ability to express abstractly what is needed. 
** The ability to make that concrete in a policy.

**Comment by britram** [comment]
I'd also like to see these things. I don't think we're there yet, and I don't think we'll get there this week. :) Pushing to post-London.

**Comment by philsbln** [comment]
I agree that these kinds of Intents/Profiles are useful. I would argue that these should go along with configuration rather than being used as parameter in the Transport API. Still there has to be some way to match different profiles within an application to the configuration.

**Comment by britram** [comment]
To discuss in person in Montreal
